dp-filter-api.nomad

```diff
@@ -0,0 +1,100 @@
+job "dp-filter-api" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  // Make sure that this API is only ran on the publishing nodes
+  constraint {
+    attribute = "${node.class}"
```

Comment:
Fixed this after speaking with Lloyd - for services that run on both web and publishing the constraint needs to be inside the group for both web and publishing groups.